administration/bundles.md

```diff
@@ -52,13 +52,13 @@ openhab> bundle:list
 Bundles are named according to the following convention:
 
 ```text
-<prefix>-<type>-<id>
+<prefix>.<type>.<id>
```

Comment:
Are you sure about this?

Whe i use `feature:list` for example, the result will be shown with dashes.

Reply:
> Bundles are named according to the following convention:

Features names follow a different convention as bundle names.